seed/static/seed/js/controllers/program_setup_controller.js

```diff
@@ -75,14 +77,22 @@ angular.module('BE.seed.controller.program_setup', []).controller('program_setup
       if (!$scope.fields.end_year) {
         $scope.errors.push('A compliance period end year (XXXX) is required!');
       }
-      let has_energy_metric = $scope.new_compliance_metric.actual_energy_column && $scope.new_compliance_metric.target_energy_column && $scope.new_compliance_metric.energy_metric_type;
-      let has_emission_metric = $scope.new_compliance_metric.actual_emission_column && $scope.new_compliance_metric.target_emission_column && $scope.new_compliance_metric.emission_metric_type;
-      if (!has_energy_metric && !has_emission_metric) {
+      let has_energy_metric = $scope.new_compliance_metric.actual_energy_column && $scope.new_compliance_metric.energy_metric_type;
```

Comment:
These changes were due to my mistake of accidentally pushing code direct to the 'bps-insights-development' branch instead of a separate branch. I went ahead and removed that commit from the 'bps' branch and add it to my own branch and a PR so we should probably remove these changes in your PR, right?